backend/TraderX/README.md

```diff
@@ -50,61 +45,27 @@ create database cmpe451;
 #### Step2: Configure your app to connect database
 in application.yml under resources, edit the fields properly:
 ```
-    username: root
-    password: root
+username: root
+password: root
+```
+
+#### Step3: Configure email properties 
+on `EmailServiceConfig.java`, use group email adress & password
+```
+static private final String senderAdress = "??";
+static private final String senderPassword = "??";
 ```
 
-#### Step2: Run the application in the project directory (where pom.xml exists)
+#### Step4: Run the application in the project directory (where pom.xml exists)
 ```
 mvn spring-boot:run
 ```
 
-#### Step3: Verify the application is running:
+#### Step5: Verify the application is running:
 
 ```
 curl -X GET localhost:8080/trial/public
 ```
 
 
```

Comment:
Maybe we can specify an example POST request using Postman.

Reply:
Just set the request type as POST on the upper left of the Postman Application, and that's all. 
Also, using swagger, one does not even need to use Postman.